README.md

```diff
@@ -5,29 +5,45 @@
 
 This project is built using [react-admin](https://marmelab.com/react-admin/).
 
-It needs at least Synapse v1.42.0 for all functions to work as expected!
+## Usage
+
+### Supported Synapse
+
+It needs at least [Synapse](https://github.com/matrix-org/synapse) v1.42.0 for all functions to work as expected!
 
 You get your server version with the request `/_synapse/admin/v1/server_version`.
 See also [Synapse version API](https://matrix-org.github.io/synapse/develop/admin_api/version_api.html).
 
 After entering the URL on the login page of synapse-admin the server version appears below the input field.
 
+### Prerequisites
+
 You need access to the following endpoints:
 
 - `/_matrix`
 - `/_synapse/admin`
 
 See also [Synapse administration endpoints](https://matrix-org.github.io/synapse/develop/reverse_proxy.html#synapse-administration-endpoints)
 
-## Step-By-Step install
+### Use without install
+
+You can use the current version of Synapse Admin without own installation direct
+via [GitHub Pages](https://awesome-technologies.github.io/synapse-admin/).
+
+**Note:**
+It is highly recommended to [deploy an own environment](#step-by-step-install),
+so that the admin endpoints (`/_synapse/admin`) do not have to be exposed to
+the internet.
```

Comment:
Since the app runs in the browser you have not to expose the endpoints to the internet :) 

I'm using the feature without having exposed my admin endpoints

But you cannot use localhost:XYZ/_synapse since most browsers do not allow the access to localhost for other sites (afaik)

Reply:
Yes, you are right. You can also set an IP filter in webserver. How to explain for not IT professionals?

Reply:
```suggestion
If you want to use the deployment, you have to make sure that the admin endpoints (`/_synapse/admin`) are accessible for your browser.
**Remember: You have no need to expose these endpoints to the internet but to your network.**
If you want your own deployment, follow the [Step-By-Step Install Guide](#step-by-step-install) below.
```